docs: README.md draft (#4)
JunbaeJs/junbae-mode: README.md

# Introduce junbae

> We introduce Junbae for people using the this vscode extenstion.
> We don't know the gender yet, but i think it will be a pretty cute parrot who will be raised by the person making this program.
> The name is taken from Junbae's name.

Reviewer: ```suggestion
We introduce Junbae for people using this vscode extension.
We don't know the gender yet, but I think it will be a pretty cute parrot who will be raised by the person making this program.
The name is taken from Junbae's name.
```

# Motivation

> We wanna create vscode extension program that does not effective for coding but gives you the satisfication of coding while typing your code.
> It is sure you have satisfication for visual effect but you does not get the help typing code.

Reviewer: ```suggestion
We craved to create a product, that does not affect function but gives amusement while typing text.
We are sure that you will be content with using our mode.
```

> We are referrence to vscode's PowerMode extension program which has been unmaintained since 2017.

Reviewer: It has different characters between Power Mode and our mode. cuz Power Mode's concept is `power`. 
they focused on shaking the screen, setting off fireworks ... etc. but our case, more relaxable, or calm.

There is one thing same with us, just typing effect. so, it's not necessary sentence i think.

> If you wanna get the confidence for typing code, using the Junbae-mode extenstion. We are sure you get the more and more satisfication this effects.

Reviewer: ```suggestion
If you wanna get confidence in typing text, use the Junbae Mode. We are sure you get more and more satisfaction from this effect.
```

# junbae-mode

> Junbae is caique parrot!

Reviewer: It should be below `Introduce Junbae`

> It is activated when you typing keyboard. It will dance to the beat of your typing

Reviewer: hmm... It's not correct actually. because Junbae Mode is activated right after vscode start

> It is activated when you typing keyboard. It will dance to the beat of your typing
> If you wanna get the source code, you visit this github url and you can contribute for the this code if you follow CONTRIBUTING.md file up.
> We hope that many people contribute the visual effect extension program for visual coding and have fun during code. Junbae blesses you!!

Reviewer: ```suggestion
It is activated when you type. It will dance to the beat of your typing
If you wanna get the source code, you visit this GitHub URL and you can contribute to this code if you follow CONTRIBUTING.md file up.
We hope that many people contribute to the visual effect extension program for visual coding and have fun during code. Junbae blesses you!!
```

```
https://github.com/JunbaeJs/junbae-mode
```

<!-- 한국어 -->

# 소개합니다 준배

이 vs extension을 사용하는 사람을 위해 준배를 소개합니다
우리는 아직 성별은 모르지만 이 프로그램을 만든 사람에 의해서 키워질 것입니다.
이름은 준배의 이름에서 따온 이름입니다.

# 동기

우리는 코딩에는 효과적이지 않지만 코드를 작성할 동안 코딩에 만족하는 vscode 확장 프로그램을 만들고 싶습니다.
시각적 효과 만족에는 확신을 하지만 코딩 작성시에는 도움이 안됩니다.
우리는 2017년 부터 관리가 안되고 있는 vscode의 PowerMode 확장 프로그램을 참조했습니다.
코딩작성의 자신감을 가지고 싶다면 준배 확장 프로그램을 사용하시고, 이 효과로부터 더욱 더 만족을 가진다고 확신합니다.

# junbae-mode

준배는 카이큐 앵무새다
키보드를 칠 때마다 활성화 됩니다. 타이핑 비트에 따라 춤을 춘다.
소스 코드를 원한다면 아래 github 주소에 방문하고 CONTRIBUTING.md 파일을 따른다면 기여할 수 있습니다.
많은 분들이 시각효과 확장프로그램 기여하고 코드하는 동안 재미를 가졌으면 합니다. 준배가 너를 축복한다!!

```
https://github.com/JunbaeJs/junbae-mode

> ```

Reviewer: 번역기 사용하시고 조금 고쳐주세요... 그리고 한국어 문서는 따로 md파일 만드는게 좋습니다.
`README-ko.md`로 만드는게 좋을 것 같습니다.
